src/rust/metric-forwarder/src/accumulate_metrics.rs

```diff
@@ -313,7 +317,7 @@ mod tests {
                 );
                 Ok(())
             }
-            _ => Err("must be exactly 2 metrics".to_string()),
+            invalid => Err(format!("Expected exactly 2 metrics, but got {:?}", invalid)),
```

Comment:
It occurs to me that it would be a bit more idiomatic to cast these failures in terms of the `panic!` macro:
```rust
invalid => panic!("Expected exactly 2 metrics, but got {:?}", invalid);
```